vsphere/tests/test_api.py

```diff
@@ -45,7 +46,7 @@ def test_ssl_cert(realtime_instance):
         assert actual_context.protocol == ssl.PROTOCOL_TLS
         assert actual_context.verify_mode == ssl.CERT_REQUIRED
         assert actual_context.check_hostname is True
-        load_verify_locations.assert_called_with(capath='/dummy/path')
+        load_verify_locations.assert_called_with(cafile='/dummy/path/cafile.pem', capath='/dummy/path')
```

Comment:
Could you also create a test when `cafile` is not set, and then one where `capath` is not set?